locking/admin.py

```diff
@@ -53,21 +54,26 @@ def lock(self, obj):
         if obj.is_locked:
             seconds_remaining = obj.lock_seconds_remaining
             minutes_remaining = seconds_remaining/60
-            locked_until = _("Still locked for %s minutes by %s") \
```

Comment:
If the `ugettext` calls are no longer needed, I don't think we need the import anymore.

Reply:
We do not need those ugettext calls. I'll get rid of the import.